quickstart_lists/core_deps_x86_64-linux_stable

```diff
@@ -0,0 +1,305 @@
+core/acbuild
```

Comment:
How valuable is it to sync _everything_?  This list will have to be updated any time core-plans accepts an additional package, which is somewhat infrequent fortunately so as a starting point this is fine, but we should be aware that this list (and windows/kernel2 equivalent) has potential for drift.